### ISSUE TITLE
E1 - Events listing filtered by keyword nests H2 events headings under the main H2 heading

### ISSUE THREAD

**Issue by mmafe** [ccswbs/hjckrrh#587]
![image](https://cloud.githubusercontent.com/assets/1927902/20580132/ae006e50-b19e-11e6-89a3-04847850050b.png)

Events related to <keyword> listing page nests H2 headings under the main H2 heading. The headings nested under the Events related to <keyword> H2 should be H3 headings.

You can get to this page by navigating to /events/upcoming/<term ID>

Note that the screenshot is from a client development site, not the live arts production site.

**Comment by mmafe** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [(5) #587 Events listing by keyword nests H2 under H2](https://trello.com/c/ZzxAQQUN/2026-5-587-events-listing-by-keyword-nests-h2-under-h2)

**Comment by ushenoyuog** [comment]
Seems to have been mergerd, closing issue.
